unit_test/test_spdm_requester/get_csr.c

```diff
@@ -42,7 +63,38 @@ libspdm_return_t libspdm_requester_get_csr_test_send_message(
         return LIBSPDM_STATUS_SUCCESS;
     case 0x3:
         return LIBSPDM_STATUS_SUCCESS;
+    case 0x4: {
+        const spdm_get_csr_request_t *spdm_request;
+        uint16_t requester_info_length;
+        uint16_t opaque_data_length;
+        uint8_t *opaque_data;
+        uint8_t *requester_info;
+        bool result;
+
+        /* Obtain the real spdm_request */
+        spdm_request =
+            (const spdm_get_csr_request_t *)((const uint8_t *)request +
+                                             sizeof(libspdm_test_message_header_t));
+
+        requester_info_length = spdm_request->requester_info_length;
+        opaque_data_length = spdm_request->opaque_data_length;
+
+        requester_info = (void*)((size_t)(spdm_request + 1));
+        result = libspdm_consttime_is_mem_equal(requester_info, right_req_info,

```

Comment:
Can't you use [assert_memory_equal](https://api.cmocka.org/group__cmocka__asserts.html#gac933d8b75ebd55e88ca85963721c05d9)?

Reply:
Thanks.I have changed `libspdm_consttime_is_mem_equalto` to `assert_memory_equal`